bigquery/cloud-client/query_params.py

```diff
@@ -40,28 +40,24 @@ def query_positional_params(corpus, min_word_count):
         AND word_count >= ?
         ORDER BY word_count DESC;
         """
-    query_job = client.run_async_query(
-        str(uuid.uuid4()),
-        query,
-        query_parameters=(
-            bigquery.ScalarQueryParameter(
-                # Set the name to None to use positional parameters (? symbol
-                # in the query).  Note that you cannot mix named and positional
-                # parameters.
-                None, 'STRING', corpus),
-            bigquery.ScalarQueryParameter(None, 'INT64', min_word_count)))
-
-    # Only standard SQL syntax supports parameters in queries.
-    # See: https://cloud.google.com/bigquery/sql-reference/
-    query_job.use_legacy_sql = False
-
-    query_job.begin()
+    # Set the name to None to use positional parameters (? symbol in the
+    # query).  Note that you cannot mix named and positional parameters.
+    # See: https://cloud.google.com/bigquery/docs/parameterized-queries/
+    query_params = [
+        bigquery.ScalarQueryParameter(None, 'STRING', corpus),
+        bigquery.ScalarQueryParameter(None, 'INT64', min_word_count)
+    ]
+    job_config = bigquery.QueryJobConfig()
+    job_config.query_parameters = query_params
+    job_id = str(uuid.uuid4())
+    query_job = client.query(query, job_config=job_config, job_id=job_id)
+
     query_job.result()  # Wait for job to complete
```

Comment:
We should use the return value for result instead of getting the table after this.